README.md

```diff
@@ -43,4 +43,9 @@ devenv
 ```
 
 
+## Development
+### Build Scripts
+We strongly suggest you to enable shared library build by default in your package build script, because it is expected to launch a lot of applications built by `devenv` for `devenv` users, and the launched toolchain may need significant memory. Enabling shared library to build your package will provide more flexibility and saves significant memory.
```

Comment:
How about using 79-char width lines?

Reply:
I prefer 119 for modern monitor. Django adapted [the same policy](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/):

```
We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult.
```

What do you think?

Reply:
Let me use 79 first, and we could continue the discussion later on.